### DISCUSSION TITLE
Inability to access sidebar menu or message field on mobile device after recent redesign

### DISCUSSION THREAD

**Discussion by skobkin** [open-webui/open-webui#2846] [General]
# Bug Report

## Description

**Bug Summary:**
This is yet another UI/UX degradation caused by recent redesign like #2827 which makes it harder to use the UI on mobile devices.

The old way to call the sidebar wasn't very convenient, but it was working any time at least. You didn't need to scroll the entire chat to access the menu or the input field.

**Steps to Reproduce:**

- Create a chat with somewhat long history (more than one screen)
- Scroll down to see the new message field which is hidden
- Header is now hidden
- Scroll entire chat back to the top to be able to access the header
- Now new message field is unavailable

**Expected Behavior:**

Being able to easily access the menu or new message field

**Actual Behavior:**

Either header with menu button and model selector or message input field are hidden until user scrolls the entire chat to the opposing side.

## Environment

- **Open WebUI Version:** 0.2.4

- **Operating System:** Android
- **Browser (if applicable):** Vivaldi

## Reproduction Details

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.

## Logs and Screenshots

**Screenshots (if applicable):**

https://github.com/open-webui/open-webui/assets/967576/c4483e54-8173-4e02-9c5e-f2e3e47b3052

## Installation Method

Docker Compose

**Comment by justinh-rahb** [comment]
What browser is that? Chrome on Android does not exhibit this behavior. When the input box is visible, yes the header gets hidden, but if I hide the input box, the header reappears immediately. This is far more usable than the old way with the nearly-invisible pill button.

**Reply by skobkin** [reply]
As I wrote earlier, it's Vivaldi on Android.

> if I hide the input box

It works the same for me. To hide the input box though, I must scroll all the way to the top of the dialog.

**Comment by tjbck** [comment]
I'll do my best to address both of your issues. I understand your frustration, however, let's keep the conversation nice, as your current tone makes it difficult to engage productively. Thanks.

**Reply by skobkin** [reply]
Sure. Thanks!

**Comment by tjbck** [comment]
Fixed on dev
![IMG_7E536A9F3FA3-1](https://github.com/open-webui/open-webui/assets/25473318/b09e2318-7f11-482f-a7b2-1cfcea2991f4)
